README.md

```diff
@@ -1,4 +1,7 @@
 # Implementing a toy ERGM from scratch in R
+This package allows to use a large step sampler in MCMC suggested by Mele (2017). 
+I just added this (ANGELO)
+
```

Comment:
@meleangelo 
I think these should be placed after the paragraph `This trial is motivated by ...` ?
And you don't have to add `I just added this (ANGELO)`, since I definitely know 😄

Reply:
It was just for fun :)